.github/workflows/release.yaml

```diff
@@ -0,0 +1,34 @@
+on:
+  push:
+    branches:
+      - master
```

Comment:
Change `master` to `aws-lambda`

Change the PR to go into `aws-lambda`.

Master does not have the lambda code, it has a different code that runs without lambda, we still have to update that code with the changes to use the assume role when calling the get parameter, i'll do that later.